notebooks/data-sources/github/thoth_PR_EDA.ipynb

```diff
@@ -0,0 +1,1892 @@
+{
```

Comment:
<p>small nit, I think just adding <code>index_col=0</code> to read_csv would get rid of this unnecessary "unnamed: 0" column :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/490/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>